draft-ietf-oauth-identity-chaining.md

```diff
@@ -229,6 +241,10 @@ The authorization server MUST validate the JWT authorization grant as specified
 
 The authorization server responds with an access token as described in section 5.1 of {{RFC6749}}.
 
+The following process rule applies:
+
+* If the JWT authorization grant in the access token request contains a "requested_cnf" claim, the authorization server MUST include the "cnf" claim from the "requested_cnf" claim in the returned token.
```

Comment:
Pieter suggested to add a metadata field to authorization server metadata that indicates if sender constraint tokens are supported or not. 

@PieterKas would that field be general about "cnf" is supported or should it rather be "assertion with requested_cnf is supported"?